src/i18n/zh.js

```diff
@@ -137,6 +137,45 @@ export default {
     },
     // 订单
     order: {
+      order1:'超过十分钟可获得积分补偿',
```

Comment:
'order1' should be 'orderCompensation'
'order2' should be 'fillOrder'
'order3 should be 'chooseShippingAddress'
'order4' should be 'deliveryTime'
'order5' should be delete because the value is same as 'order1'
'order6' should be 'total'
'order7' should be 'piece'
'order8' should be 'weChatPay'
'order9' should be 'alipay'
'order10' should be 'antsSpendBai'
........

Reply:
> The key is not standard.Please follow the naming convention.

yes sir

Reply:
> 'order1' should be 'orderCompensation'
> 'order2' should be 'fillOrder'
> 'order3 should be 'chooseShippingAddress'
> 'order4' should be 'deliveryTime'
> 'order5' should be delete because the value is same as 'order1'
> 'order6' should be 'total'
> 'order7' should be 'piece'
> 'order8' should be 'weChatPay'
> 'order9' should be 'alipay'
> 'order10' should be 'antsSpendBai'
> ........

yes sir